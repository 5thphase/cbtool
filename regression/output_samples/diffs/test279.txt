|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1406644447 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1406644490 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1406644508 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
