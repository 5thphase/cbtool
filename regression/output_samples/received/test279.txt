Background operation execution success. Operation "vm-capture" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.1.250,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,timeout:240,host:172.16.1.250,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,oldest,none --operation=vm-capture --cn=TESTCLOUD --uuid=1EAB25B8-FDA2-5B9F-8EFD-2C6C05272EE9 --daemon". The process id is 18903.

Going to unconditionally wait for 25s (25 seconds). The command line interface will be blocked during the waiting.
Waited 10 seconds... (40.0%)
Waited 20 seconds... (80.0%)
Waited 25 seconds... (100.0%)
Waited for 25 seconds.
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |147.217.82.254  |1396711187     |none                                    |9903A609-28CB-52EA-A44C-5D1883BD08CE
|ai_4      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |192.7.99.62     |1396711227     |none                                    |09555B26-C0BA-54DB-BA4E-0D14B5627636
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |73.250.19.169   |1396711245     |none                                    |79085F3C-99A6-5807-9376-0F23C78DAC03

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_33         |db2                 |platinum64  |58.227.217.212  |simhostc1                  |SUT                 |ai_3    |none       |5F21E651-D899-53E7-99B6-40882F0E1225
|vm_35         |was                 |iron32      |231.249.217.253 |simhostb1                  |SUT                 |ai_3    |none       |F70BCABD-CD09-57A5-AFD6-011B46782513
|vm_34         |client_daytrader    |micro32     |147.217.82.254  |simhostd1                  |LG                  |ai_3    |none       |677B851E-1A68-59FB-A2B2-A2D323126434
|vm_36         |client_daytrader    |micro32     |192.7.99.62     |simhostd1                  |LG                  |ai_4    |none       |A56EB9B7-C43A-5E94-90AD-E8F6950B2E65
|vm_37         |was                 |iron32      |161.229.45.43   |simhostc1                  |SUT                 |ai_4    |none       |3152CA65-B681-58B9-A6DF-AD94FEFF1982
|vm_38         |db2                 |platinum64  |153.128.206.131 |simhostc1                  |SUT                 |ai_4    |none       |8DBFB411-AA3B-5089-B4A8-FB3DACB8B492
|vm_39         |netclient           |platinum64  |73.250.19.169   |simhostb1                  |SUT                 |ai_5    |none       |99A818AE-CEF7-5CF3-B380-B83789A60F58
|vm_40         |netserver           |iron32      |92.138.222.180  |simhostc1                  |SUT                 |ai_5    |none       |5320C000-9824-5408-9205-436B6B3ACFA4

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |3.26M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |507858                        
|Total Connections Received                           |10509                         
|Total Commands Processed                             |644105                        
|Number of Keys                                       |487                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |1511424                       
|Data Size                                            |234096                        
|Index Size                                           |294336                        
|Average Object Size                                  |440.858757062                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |8                             
|AIs                                                  |3                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |8                             
|VMs ARRIVED                                          |43                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |3                             
|AIs ARRIVED                                          |7                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |180                           

 ---------------- VMC-WIDE COUNTERS ----------------
|49282BA1-6D05-5C43-9AF0-FFE568AFD5F9 (simzone_b) VM RESERVATIONS|2                             
|F04BDEFB-2E30-560C-802C-A1FABC5D8D0C (simzone_a) VM RESERVATIONS|0                             
|9C8B8765-2143-577F-9E2E-DEDF96009D33 (simzone_d) VM RESERVATIONS|2                             
|68F26DD5-96FF-5310-9ED9-3568A17CBF4A (simzone_c) VM RESERVATIONS|4                             

