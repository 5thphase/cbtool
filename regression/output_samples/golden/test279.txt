Background operation execution success. Operation "vm-capture" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.0.4,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,password:cbpwdZ9,timeout:240,host:172.16.0.4,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,oldest,none --operation=vm-capture --cn=TESTCLOUD --uuid=1EB9560C-70E8-5377-98EE-D0F6FB573F82 --daemon". The process id is 27979.

Going to unconditionally wait for 25s (25 seconds). The command line interface will be blocked during the waiting.
Waited 10 seconds... (40.0%)
Waited 20 seconds... (80.0%)
Waited 25 seconds... (100.0%)
Waited for 25 seconds.
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |237.43.51.52    |1454604108     |none                                    |79D338F8-C126-5F7A-9CB0-62065B68A72E
|ai_4      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |252.151.62.169  |1454604153     |none                                    |13B56B42-FEC0-579A-B13F-4CB0FC1CAC51
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |197.159.78.102  |1454604170     |none                                    |F3C578E2-9DCE-5C5C-9C7D-B084F3A152F5

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool      |netname    |ai      |aidrs      |uuid
|vm_33         |client_daytrader    |micro32     |237.43.51.52    |simhostb11                 |LG            |private    |ai_3    |none       |E3447E29-5A1A-5D72-89EA-AC4A79C673CF
|vm_34         |db2                 |platinum64  |164.225.113.117 |simhostd6                  |SUT           |private    |ai_3    |none       |4C71F2E8-4CA9-5C55-8BC5-6F690AD2EA15
|vm_35         |was                 |iron32      |63.160.70.192   |simhostd11                 |SUT           |private    |ai_3    |none       |A3A3F464-3FC5-5428-A6C5-1915F98D4E12
|vm_36         |db2                 |platinum64  |110.43.189.80   |simhostc6                  |SUT           |private    |ai_4    |none       |C28D9D54-9E2C-5D3D-9F2A-5806E8E1AE62
|vm_37         |client_daytrader    |micro32     |252.151.62.169  |simhosta9                  |LG            |private    |ai_4    |none       |06938716-CC7E-503B-AB71-FDE2C6EEF5E7
|vm_38         |was                 |iron32      |251.231.129.156 |simhostd12                 |SUT           |private    |ai_4    |none       |6900D46A-D162-51E2-BEC4-27A3F0693B05
|vm_39         |netclient           |platinum64  |197.159.78.102  |simhostb2                  |SUT           |private    |ai_5    |none       |6E731E4D-2F3E-54A4-AEB6-447D5D81BA93
|vm_40         |netserver           |iron32      |36.229.203.248  |simhostb5                  |SUT           |private    |ai_5    |none       |16B77D6A-03F7-503A-89DA-A28636CC0EA8

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |18.99M                        
|Redis Version                                        |2.8.6                         
|Uptime (in seconds)                                  |12856397                      
|Total Connections Received                           |548556                        
|Total Commands Processed                             |42421895                      
|Number of Keys                                       |788                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |28                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |8                             
|AIs                                                  |3                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |8                             
|VMs ISSUED                                           |46                            
|VMs ARRIVED                                          |43                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |1                             
|AI RESERVATIONS                                      |3                             
|AIs ISSUED                                           |7                             
|AIs ARRIVED                                          |7                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |300                           

 ---------------- VMC-WIDE COUNTERS ----------------
|A168E857-FDCA-5E02-98BB-32D5CA3934A5 (simzone_d) VM RESERVATIONS|2                             
|2A9C7A1E-D5EE-5CFE-BE56-2B4E298B71F7 (simzone_a) VM RESERVATIONS|1                             
|FB2A884A-73A4-5C91-B913-9AD4AF1FB28A (simzone_c) VM RESERVATIONS|3                             
|073C9CDF-01E7-5AA9-B92E-02C6DA6E75D6 (simzone_b) VM RESERVATIONS|2                             

